### PR TITLE
Gem: only include necessary files in the gem

### PR DESCRIPTION
**Pull request by jmoody**
### Motivation

Resolves **Gem contains many unnecessary files** #651

To the reviewer, I left out these files:

* ruby-gems/.calabash_settings
* ruby-gems/lib/calabash-android/canned_steps.md
* ruby-gems/lib/calabash-android/removed_actions.txt

I think Tobias is the obvious reviewer.

### CODE REVIEW

File: ruby-gem/calabash-android.gemspec [hunk 1]
```diff
@@ -11,9 +11,20 @@ Gem::Specification.new do |s|
...
+      # This should be in lib/calabash-android somewhere
+      ["irbrc"] +
+      Dir.glob("bin/**/*.rb") + ["bin/calabash-android"] +
+      Dir.glob("test-server/calabash-js/src/*.js") +
+      Dir.glob("lib/**/*.jar") +
+      Dir.glob("features-skeleton/**/*.*") +
+      ["epl-v10.html", "LICENSE"] +
+      ["lib/calabash-android/lib/TestServer.apk",
+       "test-server/AndroidManifest.xml",
+       "test-server/build.xml"]
```
Comment (sapieneptus) [inline ruby-gem/calabash-android.gemspec]: @jmoody If we remove the test server from the repo, these xml files won't be there.

**Comment by jmoody** [comment]
**REBASED** Fri Dec 18 9:50 CET

File: ruby-gem/calabash-android.gemspec (hunk 1 above)
Comment (jmoody) [inline ruby-gem/calabash-android.gemspec]: @sapieneptus I believe these .xml files are required at test _runtime_?

Are they not part of the resigning process?

@TobiasRoikjer Can you resolve this and merge?

**Comment by TobiasRoikjer** [comment]
@jmoody Can I merge this?

**Comment by jmoody** [comment]
@TobiasRoikjer It is assigned to you.  It does not have WIP in the title.  Merge if you think it is ready.

**Pull request merged**